src/Bundle/Controller/ResourceController.php

```diff
@@ -212,6 +212,9 @@ public function createAction(Request $request): Response
 
             return $this->redirectHandler->redirectToResource($configuration, $newResource);
         }
+        if ($request->isMethod('POST') && $form->isSubmitted() && !$form->isValid()) {
```

Comment:
Small improvement - we could add a blank line before if and extract the content of the check to the separate method

```suggestion

        if ($this->isFormInValid($request, $form)) {
```